### ISSUE TITLE
Support for CLJC files

### ISSUE THREAD

**Issue by lorddoig** [noprompt/lein-garden#36]
I'm not sure if you're deliberately holding off on Clojure 1.7 support until it lands officially, but in using the release candidates it's been useful to me to keep styles in CLJC files so they can be pulled directly into ClojureScript components as well as used to generate stylesheets.  Unfortunately this doesn't work with lein-garden, which will run happily enough but won't pickup changes in auto-build mode.

**Comment by noprompt** [comment]
I'm not deliberately holding off I just haven't had a chance to get to it yet. Garden has a similar issue  already open [here](https://github.com/noprompt/garden/issues/77). I'd be willing to bump the version of Clojure for this plugin though if it means the support will be available for CLJC.

**Comment by noprompt** [comment]
@lorddoig Could you investigate this and get back to me with more information or patch?

**Comment by lorddoig** [comment]
Yeah, I suspect it's probably in `ns-tracker` as I encountered an almost identical issue with another namespace tracking lib.  Nothing obvious jumping out at me yet.  I'll get back to you when I've had a chance to look properly.

**Comment by den1k** [comment]
Any updates on this? I'd love for this to work!

**Comment by noprompt** [comment]
@den1k I'm waiting for a patch at this point.

**Comment by acron0** [comment]
Bitten by this today :( Except it won't see my CLJC files at all.

**Comment by k2052** [comment]
`ns-tracker` works with `cljc` now so it should be possible to get this working. I did a quick version bump of `ns-tracker` and got FileNotFoundException's, so it isn't working out of the box, but I'll try to dive in at some point and see if I can figure out a patch.

If anyone reading this issue needs work arounds you can use [garden-watcher](https://github.com/plexus/garden-watcher) and just change line 18 to `cljc` https://github.com/plexus/garden-watcher/blob/7d241e5234a9ec57dee8af03706fd45dc04ac37b/src/garden_watcher/core.clj#L18. And if you are using boot, [boot-garden](https://github.com/martinklepsch/boot-garden) supports cljc files.

**Comment by noprompt** [comment]
@k2052 you wanna dump the stack trace here?
